### PR TITLE
Fixed typo mistakes.

### PR DESCRIPTION
**Pull request by yogeshsuhagiya**
## Description
Fixed typo mistakes.

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have linked an issue to this PR.
- [ ] I have indicated the change type and relevant package(s).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All CI checks are green (linting, build/deploy, etc).
- [ ] At least one core contributor has approved this PR.

### CODE REVIEW

**Comment by yogeshsuhagiya** [comment]
Hi @jcalcaben, 
Here is the new PR, can you please review it?

**Comment by jcalcaben** [comment]
Thanks @yogeshsuhagiya! I will put it in my backlog and will process it soon.

File: .npmrc [hunk 1]
```diff
@@ -1 +1 @@
-save-prefix="~"
```
Comment (sirugh) [inline .npmrc]: I swear we keep going back and forth on this. @zetlen sup?!

**Comment by jcalcaben** [comment]
Please remove the following commits from this PR:
* 7c93cdc74cfd57587f71589d796578c970e277c2
* 94610e0a1dea450f0f180196382052f525b3a26d
* ce472b3622174c21a1a93f086166f5ebaa40fe37

**Comment by supernova-at** [comment]
> Please remove the following commits from this PR:

That was partially my fault. Fixed and pushed 👍

**Comment by yogeshsuhagiya** [comment]
Hi @jcalcaben, is this PR now eligible for merge?

**Comment by yogeshsuhagiya** [comment]
Hi @jcalcaben, is there anything else that requires from my side?

**Pull request merged**